test/test_type_promotion.py

```diff
@@ -218,6 +218,23 @@ def test_create_bool_tensors(self):
         # this seems like odd behavior but ints also create float tensors, numpy doesn't have this function.
         self.assertEqual(torch.scalar_tensor(False, device=self.device), torch.tensor(0., device=self.device))
 
+    def test_can_cast(self):
+        self.assertTrue(torch._can_Cast(torch.int32, torch.int64))
```

Comment:
the function we'll want to expose in python should be `can_cast` without the capital letter or leading underscore